docs/articles/documentation/using-testcafe/programming-interface/index.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: docs
+title: TestCafe Programming Interface
+permalink: /documentation/using-testcafe/programming-interface/
+---
+# TestCafe Programming Interface
+
+This section provides reference for the API that allows you use TestCafe from your NodeJS applications.
+
+#### Point of Entry
+
+* the [createTestCafe](/testcafe/documentation/using-testcafe/programming-interface/module-level-members/#createTestCafe) factory function
+
+#### Contents
```

Comment:
I don't like this separation on `Point of Entry` and `Contents`. Maybe `Point of Entry` and `Classes` or smthg like that.